lib/utils.js

```diff
@@ -64,11 +65,42 @@ const utils = {
     utils.log(`${c.redBright('Error')} ${err && err.message ? err.message : err}`, newline);
   },
 
-  loadUserConfig: async () => {
+  loadUserConfig: async (checkIsEmpty = true) => {
     const f = resolveUserConfig();
     await ensureFile(f);
     const config = yaml.safeLoad(await readFile(f)) || {};
-    if (isEmpty(config)) {
+    if (config.apiToken) {
+      const token = config.apiToken;
+      delete config.apiToken;
+      const defaultUrl = process.env.LOSANT_API_URL;
```

Comment:
this could be wrong - what happened to all the JWT stuff that actually would know for sure what url a token is from?